spynnaker/pyNN/models/neuron/synapse_io.py

```diff
@@ -118,649 +117,604 @@ def undelayed_max_words(self):
 
     @property
     def delayed_max_words(self):
-        """ Maximum number of words, excluding headers, in a row of the\
+        """ Maximum number of words, excluding headers, in a row of the
             undelayed matrix
 
         :rtype: int
         """
         return self.__delayed_max_words
 
 
-class SynapseIORowBased(object):
```

Comment:
Is there a reason the functions below this are now class-less?

Reply:
The class was already stateless, this just made sure that it was clear.  It means that you don't need to pass it round (which was pointless anyway as it had no state ;) )